views/import.erb

```diff
@@ -2,7 +2,7 @@
 
 <form action="/import" method="post">
   <div class="field has-addons">
-    <div class="control">
+    <div class="control is-required">
```

Comment:
`:path` is also required according to `lib/ronin/app/validations/import_params.rb`. Need to add `is-required` to it as well.